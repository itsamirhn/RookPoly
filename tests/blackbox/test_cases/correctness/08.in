4
1001
0101
0011
0001
