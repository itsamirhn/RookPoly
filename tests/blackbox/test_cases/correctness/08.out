11X^3 + 21X^2 + 9X + 1
